zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/approval/tasks/ReleasePopulator.java

```diff
@@ -89,6 +89,7 @@ private static void addRelatedDataset(Release release, ContentDetail contentDeta
     }
 
     private static void addRelatedAPIDataset(Release release, ContentDetail contentDetail) {
+        contentDetail.setUri(contentDetail.uri.substring(0, contentDetail.uri.indexOf("/versions")));
```

Comment:
I would expect the release populator to use the content detail as it comes rather than modify it. As far as I can tell this uri comes from the json file, should the json file not have the `/versions` path in the uri in the first place? This seems more like a hack to me

Reply:
Also it would be a problem if the url contains `/versions` in any part of the path (and not just a suffix)